src/export/Export.cpp

```diff
@@ -1519,7 +1519,7 @@ void ShowExportErrorDialog(wxString ErrorCode,
    TranslatableString message,
    const TranslatableString& caption)
 {
-   ShowErrorDialog(nullptr,
+   ShowExceptionDialog(nullptr,
```

Comment:
So I see many straightforward substitutions of one function for another.

A question is whether this is substitution is complete enough.

Reply:
I've grepped ShowErrorDialog and replaced all the places I felt made sense.